lib/l10n/intl_en.arb

```diff
@@ -88,6 +89,7 @@
   "hintWebsiteLink": "http://google.com",
   "hintEvaluation": "Final exam",
   "hintPoints": "4.0",
+  "hintSelectSources": "Filter information in the app based on who added it:",
```

Comment:
Uhm, this was actually correct before with the "Select information sources" or whatever it was. It's used on the page to prompt the user to mark the checkboxes.

Reply:
Ok